packages/overlays/stories/index.stories.mdx

```diff
@@ -0,0 +1,646 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { renderLitAsNode } from '@lion/core';
+import {
+  OverlayMixin,
+  withBottomSheetConfig,
+  withDropdownConfig,
+  withModalDialogConfig,
+} from '../index.js';
+
+import './lion-demo-overlay.js';
+import './applyDemoOverlayStyles.js';
+
+<Meta title="Overlay System/System" parameters={{ component: 'lion-demo-overlay' }} />
+
+# Overlays System
+
+Supports different types of overlays like dialogs, toasts, tooltips, dropdown, etc.
```

Comment:
Maybe we can do less 'telegram style' here and do a summary of this file: https://github.com/ing-bank/lion/blob/master/packages/overlays/docs/OverlaySystemScope.md and then link to the full text.
Would be nice to have an introduction that explains the goal, the scope and the unique selling points of our system. Also nice to have an exhaustive list of which components (both aria widgets as recognized by w3c and 'community ux components') can be built with our system (or were in scope when the system was designed.